tests/test_cli_compile.py

```diff
@@ -1656,3 +1656,118 @@ def test_triple_equal_pinned_dependency_is_used(
     assert out.exit_code == 0, out
     for line in out_expected_content:
         assert line in out.stderr
+
+
+@pytest.mark.network
+@pytest.mark.parametrize(
+    ("fname", "content"),
+    (
+        # setuptools
+        (
+            "setup.cfg",
+            """
+                [metadata]
+                name = sample_lib
+                author = Vincent Driessen
+                author_email = me@nvie.com
+
+                [options]
+                packages = find:
+                install_requires =
+                    small-fake-a==0.1
+                    small-fake-b==0.2
+
+                [options.extras_require]
+                dev =
+                    small-fake-c==0.3
+                    small-fake-d==0.4
+                test =
+                    small-fake-e==0.5
+                    small-fake-f==0.6
+            """,
```

Comment:
```suggestion
            dedent(
                """\
                [metadata]
                name = sample_lib
                author = Vincent Driessen
                author_email = me@nvie.com

                [options]
                packages = find:
                install_requires =
                    small-fake-a==0.1
                    small-fake-b==0.2

                [options.extras_require]
                dev =
                    small-fake-c==0.3
                    small-fake-d==0.4
                test =
                    small-fake-e==0.5
                    small-fake-f==0.6
                """
            ),
```

dedent could help to reduce extra indentations

Reply:
Sorry, what do you mean? I can drop one level of indentation from the multiline string without moving `dedent`, Python is liberal about it. It's up to personal preferences, I've chosen to add the indentation to match how we indent the code inside braces. See https://github.com/wemake-services/wemake-python-styleguide/issues/1025 for more context. However, if you prefer another style, I can change it, not a big deal.

Reply:
> I can drop one level of indentation from the multiline string without moving dedent, Python is liberal about it.

Yeah, but the result files will be indented too wide. In `pip-tools` we use `dedent` in that cases. See references:
- [1](https://github.com/jazzband/pip-tools/blob/37946f212248e932d5f7022f7fcc8810ba5d8cd2/tests/test_cli_compile.py#L109-L113)
- [2](https://github.com/jazzband/pip-tools/blob/37946f212248e932d5f7022f7fcc8810ba5d8cd2/tests/test_cli_compile.py#L810-L823)
- [3](https://github.com/jazzband/pip-tools/blob/37946f212248e932d5f7022f7fcc8810ba5d8cd2/tests/test_cli_compile.py#L931-L935)
- [4](https://github.com/jazzband/pip-tools/blob/37946f212248e932d5f7022f7fcc8810ba5d8cd2/tests/test_cli_compile.py#L1062)

Reply:
Oh, I didn't notice that `dedent` is already there:

```
stream.write(dedent(content))
```

Reply:
Then, it's okay. My bad.